prod-aos-rcar.yaml

```diff
@@ -0,0 +1,379 @@
+desc: "Xen-Troops development setup for Renesas RCAR Gen3 hardware"
+min_ver: "0.8"
+
+variables:
+  YOCTOS_WORK_DIR: "yocto"
+  DOM0_BUILD_DIR: "build-dom0"
+  DOMD_BUILD_DIR: "build-domd"
+  DOMF_BUILD_DIR: "build-domf"
+  XT_DOMD_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-domd.dtb"
+  XT_DOMF_DTB_NAME: "salvator-generic-domu.dtb"
+  XT_XEN_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-xen.dtb"
+  XT_OP_TEE_FLAVOUR: "generic_dt"
+
+common_data:
+  # Sources used by all yocto-based domains
+  sources: &COMMON_SOURCES
+    - type: git
+      url: "git://git.yoctoproject.org/poky"
+      rev: dunfell
+    - type: git
+      url: "git://git.openembedded.org/meta-openembedded"
+      rev: dunfell
+    - type: git
+      url: "git://git.yoctoproject.org/meta-virtualization"
+      rev: dunfell
+    - type: git
+      url: "https://github.com/xen-troops/meta-xt-common.git"
+      rev: master
+    - type: git
+      url: "https://github.com/xen-troops/meta-xt-rcar.git"
+      rev: master
+    - type: git
+      url: "https://github.com/al1img/meta-xt-prod-aos-rcar.git"
+      rev: aos_integration
+
+  # Common configuration options for all yocto-based domains
+  conf: &COMMON_CONF
+    - [SSTATE_DIR, "${TOPDIR}/../../../common_data/sstate"]
+    - [DL_DIR, "${TOPDIR}/../../../common_data/downloads"]
```

Comment:
Please use `${TOPDIR}/../common_data` to avoid creating folders outside the tree.

Reply:
This idea was taken from https://github.com/xen-troops/meta-xt-prod-devel-rcar-gen4.
I think it makes sense to have it outside. I've been rebuilt the product few times from scratch because cache is not outside.

Reply:
I'm OK with such approach, and I'm using the same `../../../` to speed up my development activities, but this may be a surprise for a customer to see the new folder `common_data` on the same level as `my_aos_build`.
In general, I'm OK.